### PR TITLE
New Presence: Honeygain

### PR DESCRIPTION
**Pull request by chaos511**
Capture

### CODE REVIEW

File: Honeygain/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "service": "Honeygain",
+    "description": {
+      "en": "Honeygain is a crowdsourced network company which enables other businesses to conduct web intelligence, market research and content delivery."
+    },
+    "url": "dashboard.honeygain.com",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/fjaT2cF.jpg",
+    "thumbnail": "https://i.imgur.com/pThjE43.png",
+    "color": "#FFFF00",
+    "tags": [],
```
Comment (frozaum) [inline Honeygain/dist/metadata.json:15]: add some tags

File: Honeygain/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    },
+    "service": "Honeygain",
+    "description": {
+      "en": "Honeygain is a crowdsourced network company which enables other businesses to conduct web intelligence, market research and content delivery."
+    },
+    "url": "dashboard.honeygain.com",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/fjaT2cF.jpg",
+    "thumbnail": "https://i.imgur.com/pThjE43.png",
+    "color": "#FFFF00",
```
Comment (frozaum) [inline Honeygain/dist/metadata.json:14]: minor thing, not necessary: Can you set the color of the logo in HEX?

File: Honeygain/dist/metadata.json (hunk 1 above)
Comment (chaos511) [reply Honeygain/dist/metadata.json]: Tags Added

File: Honeygain/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    "service": "Honeygain",
+    "description": {
+      "en": "Honeygain is a crowdsourced network company which enables other businesses to conduct web intelligence, market research and content delivery."
+    },
+    "url": "dashboard.honeygain.com",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/fjaT2cF.jpg",
+    "thumbnail": "https://i.imgur.com/pThjE43.png",
+    "color": "#fad63a",
+    "tags": ["money","passive","bearmoney","free"],
```
Comment (Bas950) [inline Honeygain/dist/metadata.json:15]: add a space after the comma of a tag
`["money", "passive", "bearmoney", "free"]`

File: Honeygain/dist/presence.js [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    }else if(path.includes("/referrals")){
+        presenceData.details="Viewing Referrals"
+        try{
+            presenceData.state="Referral Code: "+document.getElementsByClassName("sc-RefOD izklIM sc-bwCtUz gqfLZQ")[0].value.split("https://r.honeygain.money/")[1]
+        }catch(e){}
+    }else if(path.includes("/faq")){
+         presenceData.details="Viewing FAQ"
+    }else if(path.includes("/profile")){
+        presenceData.details="Viewing Profile settings"
+   }
```
Comment (Bas950) [inline Honeygain/dist/presence.js:46]: If you use VSCode I suggest you add the Prettier extension cause your code has no spaces xD

File: Honeygain/dist/presence.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    //Grab and process all your data here
+
+    // element grabs //
+    // api calls //
+    // variable sets //
+}
+setInterval(10000, myOutsideHeavyLiftingFunction);
+//Run the function seperate from the UpdateData event every 10 seconds to get and set the variables which UpdateData picks up
+
+*/
```
[the remark this answers is not in the document]
Comment (chaos511) [reply Honeygain/dist/presence.js]: Done

File: Honeygain/dist/metadata.json (hunk 3 above)
Comment (chaos511) [reply Honeygain/dist/metadata.json]: Space Added

File: Honeygain/dist/presence.js (hunk 4 above)
Comment (chaos511) [reply Honeygain/dist/presence.js:32]: I will look into that 👍

**Comment by chaos511** [comment]
> Add support for the main page.
@doomlerd what do you mean by the main page are you saying honeygain.com instead of dashboard.honeygain.com?

**Comment by doomlerd** [comment]
> > Add support for the main page.
> 
> @doomlerd what do you mean by the main page are you saying honeygain.com instead of dashboard.honeygain.com?

I meant both.

**Pull request merged**